stage-0-proposals.md

```diff
@@ -15,6 +15,7 @@ Stage 0 proposals are either
 |   | [Function bind syntax][bind-syntax]                                        | Brian Terlson & Matthew Podwysocki   | 0     |
 |   | [Function expression decorators][func-expr-decorators]                     | Igor Minar                           | 0     |
 |   | [`is{Type}` methods][is-types]                                             | James M Snell                        | 0     |
+|   | [JSON superset][json-superset]                                             |                                      | 0     |
```

Comment:
to be on the list, it would need identified champion(s); typically this would be done before making this PR.